json_schemas/services-g-cloud-7-iaas.json

```diff
@@ -444,7 +444,10 @@
       "type": "object",
       "properties": {
         "value": {
-          "type": "number"
+          "type": "number",
+          "minimum": 0,
+          "maximum": 100,
+          "exclusiveMaximum": true
```

Comment:
Why do we not allow 100%? Are there any existing services that would fail this validation?